Makefile.PL

```diff
@@ -124,6 +124,7 @@ configure_requires 'Module::Install::XSUtil';
 test_requires 'JSON::PP';
 test_requires 'Test::Fatal';
 test_requires 'Test::More' => 1.302015;
+test_requires 'MIME::Base32';
```

Comment:
I guess that means that the installation instructions must be updated too.

Reply:
On the other hand it also says 

>If you have any issue with installation, and installed with `cpanm`, redo the installation above but without the `--notest` and with the `--verbose` option. Installation will take longer time.
https://github.com/zonemaster/zonemaster/blob/master/docs/public/installation/zonemaster-engine.md#troubleshooting-installation

We also have in internal documents under https://github.com/zonemaster/zonemaster/tree/master/docs/internal/distrib-testing that would need to be updated.

Reply:
I’ve checked the documents you linked and I think they don’t need to be updated.

To be absolutely sure, I installed `MIME::Base32` from my system. I ran `cpanm --notest .`, then `cpanm .`. In both cases, `MIME::Base32` is installed automatically by `cpanm` (even though, in the first case, it isn’t required, which is rather odd).